### ISSUE TITLE
[F-Droid] Installation fails - libtermux.so not page-aligned

### ISSUE THREAD

**Issue by conrad-heimbold** [termux/termux-float#4]
Hi, I just tried to install termux-float via F-Droid (privileged extension installed), but I am getting the following adb logcat error: 
```
15679 15679 I LoadedApk: No resource references to update in package common
15679 15679 I LoadedApk: No resource references to update in package org.cyanogenmod.hexolibre
15679 15679 I LoadedApk: No resource references to update in package ovh.ice.icecons
15679 15690 D DefContainer: Copying /data/user/0/org.fdroid.fdroid/files/install--1832736247.apk to base.apk
982  1061 D NativeLibraryHelper: Library 'libtermux.so' is not page-aligned - will not be able to open it directly from apk.
982  1061 W NativeHelper: Failure copying native libraries [errorCode=-2]
982  1061 I art     : Starting a blocking GC Explicit
982  1061 I art     : Explicit concurrent mark sweep GC freed 62230(3MB) AllocSpace objects, 29(708KB) LOS objects, 33% free, 18MB/28MB, paused 2.830ms total 128.478ms
2800  2800 E AppDetails: install aborted with errorMessage: Error -2: The package archive file is invalid.
```

Do you know how to fix this? 
Thanks for an answer in advance!

**Comment by ShapeShifter499** [comment]
I'm also having issues installing from F-Droid but I don't see the same error in my logcat.
[termux-float_logcat.txt](https://github.com/termux/termux-float/files/946685/termux-float_logcat.txt)

System
Moto X Pure
LineageOS 14.1 Nightly 04/10/2017
F-Droid 0.103-alpha2

**Comment by brbsix** [comment]
The APK needs to get zipaligned, e.g. `zipalign -f -p -v 4 com.termux.window_9.apk com.termux.window_9-zipaligned.apk`

microG project also had this issue with their F-droid builds, see https://github.com/microg/android_packages_apps_GmsCore/issues/280

**Comment by fornwall** [comment]
Was this fixed on the side of F-Droid?

**Comment by conrad-heimbold** [comment]
No, but because the fix is so trivial (and not termux' fault); I thought this issue should be closed.

**Comment by fornwall** [comment]
Ok, we can leave it open to track the issue on this side.

I just created an issue at F-Droid, https://gitlab.com/fdroid/fdroidclient/issues/968.

**Comment by krt16s** [comment]
This is definitely our fault, but I am still not sure what broke down. I moved @fornwall s issue ticket to the server side, see https://gitlab.com/fdroid/fdroidserver/issues/288.

**Comment by Seferi** [comment]
I'm getting the following error after trying to install from F-Droid
Failed to install due to an unknown issue.

LineageOS 14.1, Android 7.1.2, Oneplus 2

**Comment by fornwall** [comment]
This is now finally fixed in version 0.10 of Termux:Float which is now available on F-Droid. Thanks for the help fixing this!

**Comment by SickBoy-Droid** [comment]
I am not able to install termux float in my Android phone which is running on Android pie!
